models/Movie.model.js

```diff
@@ -0,0 +1,47 @@
+const { Schema, model } = require("mongoose");
+
+const movieSchema = new Schema(
+  {
+    title: {
+      type: String,
+      require: true
+    },
+    plot: {
+      type: String,
+    },
+    genre: {
+      type: String,
+    },
+    year: {
+      type: String,
+    },
```

Comment:
year should be of type Date: 

```suggestion
year: Date
```

also see: https://mongoosejs.com/docs/schematypes.html